echo-scheduler/src/main/resources/db/changelog/20190520-initial-schema.sql

```diff
@@ -0,0 +1,171 @@
+--liquibase formatted sql
```

Comment:
Orca and clouddriver have the initial schema specified in YAML; curious why in echo we're giving explicit SQL statements.

Reply:
because I got the SQL from `quartz` and I didn't want to convert it to YAML. I briefly looked if there is a tool to do the conversion but didn't find one so I opted to keep it in SQL. if you have a strong preference and know of a way to convert it to YAML I can do that

Reply:
Sounds good, no issues with leaving it as SQL, was just curious.